java/test.sh

```diff
@@ -15,23 +15,13 @@ check_style=$(mvn checkstyle:check)
 echo "${check_style}"
 [[ ${check_style} =~ "BUILD FAILURE" ]] && exit 1
 
+pushd $ROOT_DIR/..
```

Comment:
Does Bazel have the tool to check style like `mvn check`?

Reply:
this is a temporary method before we know how to check style with bazel